docs/pages/faq.mdx

```diff
@@ -94,19 +94,20 @@ Here is a detailed breakdown of the differences between Teleport's editions.
 
 ## Which version of Teleport is supported?
 
-Teleport provides security-critical support for the current and two previous
-releases. With our typical release cadence, this means a release is usually
-supported for 9 months.
+Teleport releases a new major version approximately every 4 months, and provides
+security-critical support for the current and two previous major versions. With
+our typical release cadence, we usually support each major version for 12
+months.
 
 ### Supported versions
 
 Here are the major versions of Teleport and their support windows:
 
-| Release | Release Date     | EOL          | Minimum `tsh` version|
-|---------|------------------|--------------|----------------------|
-| v13.0   | May 8, 2023      | February 2024| v12.x.x              |
-| v12.0   | Feb 6, 2023      | November 2023| v11.x.x              |
-| v11.0   | October 24, 2022 | September 2023 | v10.x.x              |
+| Release | Release Date | EOL            | Minimum `tsh` version |
+|---------|--------------|----------------|-----------------------|
+| v14.0   | Sep 20, 2023 | September 2024 | v13.0.0               |
+| v13.0   | May 8, 2023  | February 2024  | v12.0.0               |
+| v12.0   | Feb 6, 2023  | November 2023  | v11.0.0               |
```

Comment:
@zmb3 Should I update the EOL months for v13 and v12 to one year after the release dates?

Reply:
Yes, v12 would go EOL when v15 is released (January 2024), and then v13 would go EOL around May 2024.